### DISCUSSION TITLE
Allow Filter for Not completed Habits to Hide them

### DISCUSSION THREAD

**Discussion by Anshad2u** [iSoron/uhabits#1371] [Feature Requests]
I want a feature to **Allow Filter for Not completed Habits to Hide them** so that our habits list is clean and decluttered for that day and we can focus only on those habit that we can actually complete that day.
Right now we have the ability to filter only COMPLETED HABITS

**Comment by LordJon** [comment]
I'm not sure if I understand, but filtering not completed habits will only show the ones that already have been completed and don't require any action right? I'm missing why that would be useful.

Maybe this was changed in V2 (I'm stil on the old version)... But I use the hide completed habits filter so I'll only see the ones that actually can be completed that day.

**Reply by Anshad2u** [reply]
I want this

1- Filter (Hide) **Completed Habits** ✔
2- Filter (Hide) **Not Completed Habits** ❌
3- Filter (Hide) **Skipped Habits** ➖

So I Can show Only Habits with **Missing Data (i.e., ? - Question Mark)**

Right now Only Number 1 is Implemented.
If I have Option 2 & 3, I can Focus only on those habits with Missing data.
I don't need to keep seeing NOT COMPLETED HABITS (❌) throughout that Day
